src/rsz/src/Resizer.tcl

```diff
@@ -91,6 +91,38 @@ proc set_layer_rc { args } {
   }
 }
 
+sta::define_cmd_args "report_layer_rc" {[-corner corner]}
+proc report_layer_rc { args } {
+  sta::parse_key_args "report_layer_rc" args \
+    keys {-corner} \
+    flags {}
+  set corner [sta::parse_corner_or_all keys]
+  set tech [ord::get_db_tech]
+  set no_routing_layers [$tech getRoutingLayerCount]
```

Comment:
Does this need checking that a tech is loaded?

Reply:
Probably

```
openroad> report_layer_rc
invalid command name "NULL"
while evaluating report_layer_rc
```